library/systemd/src/modules/systemd_service.rb

```diff
@@ -70,12 +72,14 @@ class SystemdServiceClass < Module
     include Yast::Logger
 
     UNIT_SUFFIX = ".service".freeze
+    SERVICE_PROPMAP = SystemdUnit::DEFAULT_PROPMAP.merge(trigerred_by: "TriggeredBy")
```

Comment:
two typos: *trigerred* should be *triggered*.

Reply:
ah, thanks for catch. The second one (the string) is systemd name, so it is correct one.